frontend/src/graph/operators/mutation.gql

```diff
@@ -322,44 +322,16 @@ mutation DeleteSegment($segment_id: ID!) {
 mutation EditSegment(
 	$project_id: ID!
 	$id: ID!
-	$params: SearchParamsInput!
+	$query: String!
 	$name: String!
 ) {
-	editSegment(project_id: $project_id, id: $id, params: $params, name: $name)
+	editSegment(project_id: $project_id, id: $id, query: $query, name: $name)
 }
 
-mutation CreateSegment(
-	$project_id: ID!
-	$name: String!
-	$params: SearchParamsInput!
-) {
-	createSegment(project_id: $project_id, name: $name, params: $params) {
+mutation CreateSegment($project_id: ID!, $name: String!, $query: String!) {
+	createSegment(project_id: $project_id, name: $name, query: $query) {
 		name
 		id
-		params {
```

Comment:
Never use these on the frontend and were not ever sending them to the backend to be saved